homeassistant/components/sensor/onewire.py

```diff
@@ -109,34 +116,32 @@ def state(self):
     @property
     def unit_of_measurement(self):
         """Return the unit the value is expressed in."""
-        return TEMP_CELSIUS
+        return self._unit_of_measurement
 
     def update(self):
         """Get the latest data from the device."""
-        temp = -99
+        value = -99
```

Comment:
Default value should be `None`

Reply:
Sure, I just went with what was there. Will update.